charts/hedera-mirror-importer/values.yaml

```
@@ -95,6 +95,168 @@ resources:
+    for: 1m
+    labels:
+      severity: warning
+
+  ImporterParseErrors:
+    annotations:
+      description: "Encountered {{ $value | humanizePercentage }}% errors trying to parse {{ $labels.type }} stream files for {{ $labels.namespace }}/{{ $labels.pod }}"
+      summary: "Error rate parsing {{ $labels.type }} exceeds 5%"
+    enabled: true
+    expr: sum(rate(hedera_mirror_parse_duration_seconds_count{application="hedera-mirror-importer", success="false"}[3m])) by (namespace, pod, type) / sum(rate(hedera_mirror_parse_duration_seconds_count{application="hedera-mirror-importer"}[3m])) by (namespace, pod, type) > 0.05
+    for: 2m
+    labels:
+      severity: critical
+
+  ImporterParseLatency:
+    annotations:
+      description: Averaging {{ $value | humanizeDuration }} trying to parse {{ $labels.type }} stream files for {{ $labels.namespace }}/{{ $labels.pod }}
+      summary: Took longer than 2s to parse {{ $labels.type }} stream files
+    enabled: true
+    expr: sum(rate(hedera_mirror_parse_duration_seconds_sum{application="hedera-mirror-importer"}[3m])) by (namespace, pod, type) / sum(rate(hedera_mirror_parse_duration_seconds_count{application="hedera-mirror-importer"}[3m])) by (namespace, pod, type) > 2
+    for: 1m
+    labels:
+      severity: critical
+
+  ImporterPublishLatency:
+    annotations:
+      description: Took {{ $value | humanizeDuration }} to publish {{ $labels.entity }}s to {{ $labels.type }} for {{ $labels.namespace }}/{{ $labels.pod }}
+      summary: Slow {{ $labels.type }} publishing
+    enabled: true
+    expr: sum(rate(hedera_mirror_importer_publish_duration_seconds_sum{application="hedera-mirror-importer"}[3m])) by (namespace, pod, type, entity) / sum(rate(hedera_mirror_importer_publish_duration_seconds_count{application="hedera-mirror-importer"}[3m])) by (namespace, pod, type, entity) > 1
+    for: 1m
+    labels:
+      severity: critical
+
+  ImporterBalanceAge:
+    annotations:
+      description: Last processed balance file for {{ $labels.namespace }}/{{ $labels.pod }} was {{ $value | humanizeDuration }} ago
+      summary: Mirror Importer hasn't processed a balance file in more than 16m
+    enabled: true
+    expr: sum(rate(hedera_mirror_parse_latency_seconds_sum{application="hedera-mirror-importer",type="BALANCE"}[3m])) by (namespace, pod) / sum(rate(hedera_mirror_parse_latency_seconds_count{application="hedera-mirror-importer",type="BALANCE"}[3m])) by (namespace, pod) > 960
+    for: 1m
+    labels:
+      severity: critical
+
+  ImporterRecordAge:
+    annotations:
+      description: Last processed record file for {{ $labels.namespace }}/{{ $labels.pod }} was {{ $value | humanizeDuration }} ago
+      summary: Mirror Importer hasn't processed a record file in more than 15s
```

For this , wouldn't it be more appropriate to say
```suggestion
      summary: Mirror Importer backlogged, record stream file older than 15s
```

Granted we do want an alert if the mirror node hasn't processed any files in an extended period of time.

Seems like 2 related but different metrics I'm highlighting

1. An alert for when the age of the files being processed is over a threshold, to highlight the mirror node has fallen behind, likely due to load.
2. An alert for when the mirror node hasn't processed a file stream item within a certain threshold range of time.

#1 can only happen when steam files are received, #2 would happen when none are received.
Granted in a rare situation of delayed and pulsated stream emits both alert could be triggered. 

With the `hedera_mirror_parse_latency` latency I imagine #1 is what's possible here and #2 is a stretch that could be attempted in a future ticket.

What do you think?

---

True. I reworded it to the below, hopefully it addresses your concern.
```yaml
description: The difference between the file timestamp and when it was processed is {{ $value | humanizeDuration }} for {{ $labels.namespace }}/{{ $labels.pod }}
summary: Mirror Importer record stream processing has fallen behind
```

About 2), I think we do effectively have that already, at least for the record stream. If the record stream TPS drops to zero we have an alert. Unfortunately that metric also includes unsuccessful record items that get repeatedly parsed from bad files, but we also have parser error percentage alert that will catch that. So while it's not exactly comparing to the expected close interval it should still catch any issues.

We are missing the TPS logic for balance stream though.